### PR TITLE
UI for diet and training done.

### PR DESCRIPTION
**Pull request by christopher-jensen**
Now possible to start appending PDF files to the stacklayout trough a namespacebinding

### CODE REVIEW

File: Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/MainPage.xaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              x:Class="Club_Sportstimer.MainPage"
+			 xmlns:sfpdfviewer="clr-namespace:Syncfusion.SfPdfViewer.XForms;assembly=Syncfusion.SfPdfViewer.XForms"
```
Comment (CasperKjaerhus) [inline Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/MainPage.xaml:5]: Indentation 👎

File: Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Page/ProgramOption.xaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Club_Sportstimer.Page.ProgramOption"
+			 Title="Sportstiming">
```
Comment (CasperKjaerhus) [inline Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Page/ProgramOption.xaml:5]: indentation 👎

File: Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Page/ProgramOptions/DietPrograms.xaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Club_Sportstimer.Page.ProgramOptions.DietPrograms">
+    <ContentPage.Content>
+		<ScrollView>
```
Comment (CasperKjaerhus) [inline Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Page/ProgramOptions/DietPrograms.xaml:6]: indentation

File: Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Views/MainMenu.xaml.cs [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿using Club_Sportstimer.Page;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```
Comment (CasperKjaerhus) [inline Xamarin/Club_Sportstimer/Club_Sportstimer/Club_Sportstimer/Views/MainMenu.xaml.cs:4]: slet

**Review by CasperKjaerhus — APPROVED** [review]
Fix indentation 👍

**Review by anesheim-uni — APPROVED** [review]
Enig i Caspers comments ift. indentation, ellers fin

**Pull request merged**